package.json

```diff
@@ -65,6 +66,7 @@
     "react": "^16.6.3",
     "react-dom": "^16.6.3",
     "react-test-renderer": "^16.6.3",
+    "regenerator-runtime": "^0.13.1",
```

Comment:
Whats this? I don't need it locally

Reply:
I think you'll find you do 🤔 I tried removing it and it errored, so I added it back in.

Needed by Jest.

Reply:
Hmm running `npm run test` and `npm run build` I dont need them :/ Not sure why I dont and you do, strange

Reply:
Just double-checked - you're right! `jest` used to fail without it, but perhaps I was missing the env stuff in babel.config.js at that time.

Have now removed. Thanks!